packages/core/src/main.ts

```diff
@@ -0,0 +1,24 @@
+import { expandVars, HoprDB, PublicKey } from '@hoprnet/hopr-utils'
+import HoprEthereum from '@hoprnet/hopr-core-ethereum'
+import PeerId from 'peer-id'
+import Hopr, { HoprOptions, VERSION } from '.'
+
+export function createHoprNode(peerId: PeerId, options: HoprOptions) {
```

Comment:
I suspect you propose that anyone who wants to test can duplicate this function and create their own class instances as needed?

Reply:
Also please document the function and its intended use.

Reply:
More like this is the best way to create a `hopr` node, but yes, I’ll add documentation

Reply:
Added documentation to function, we’ll likely need to refactor it a bit more since `options` it’s more likely to be migrated to `HoprDB` rather than using `options` directly. 

@tolbrino Let me know whether you think I should do it in this PR.

Reply:
Discussed offline to do this in a separate PR, resolving